lib/util.js

```diff
@@ -41,7 +42,8 @@ exports.format = function(f) {
         return x;
     }
   });
-  for (var x = args[i]; i < len; x = args[++i]) {
+
```

Comment:
I thought this PR was focusing on the array creation portion. It's not a wholesale replace of `var`, so this seemed a little out of place.